interfaces/system_key_test.go

```diff
@@ -62,6 +64,12 @@ func (s *systemKeySuite) SetUpTest(c *C) {
 	c.Assert(err, IsNil)
 	s.buildID = id
 
+	s.seccompCompilerVersion = "123 2.3.3 abcdef123"
+	testutil.MockCommand(c, filepath.Join(dirs.DistroLibExecDir, "snap-seccomp"), fmt.Sprintf(`
```

Comment:
This is curious. What is the interaction of code using `/proc/self/exe` and tests that build a binary at a random location?

Reply:
`snap-seccomp` is expected to be located at the same location as current binary. So we mock where `/proc/self/exe` points to and then drop snap-seccomp in that directory.